tests/unit/pipeline/extractors/test_files.py

```diff
@@ -27,6 +25,17 @@ def fixture_directory():
         yield tempdir
 
 
+@pytest.fixture
+def unspported_file(fixture_directory):
```

Comment:
*supported

Reply:
Oops. I'll patch that un in a follow up PR.